content/en/contribute/code/releasing.md

```diff
@@ -16,4 +18,4 @@ Follow the [instructions in the readme](https://github.com/medic/cht-conf/#user-
 
 ## Android apps
 
-Follow the instructions in the [Android > Releasing]({{< ref "core/guides/android/releasing" >}}) section.
+Follow the instructions in the [Android > Releasing]({{< ref "contribute/code/android/releasing" >}}) section.
```

Comment:
I find it a bit confusing to have two "Releasing" pages under the handbook. Any chance to rename them or merge them into one?

Reply:
I've created a new subdirectory under "contributing > code > CHT core" so it's more obvious which is Android and which is CHT Core. I'd rather not merge them because they're very different.